src/components/AppConfig/Vector.tsx

```diff
@@ -153,16 +191,19 @@ function QdrantConfig({ settings, onChange }: Props<QdrantSettings>) {
 
 function GrafanaVectorAPIConfig({ settings, onChange }: Props<GrafanaVectorAPISettings>) {
   return (
-    <Field label="URL" description="URL of the Grafana Vector API">
-      <Input
-        width={60}
-        name="url"
-        data-testid={testIds.appConfig.grafanaVectorApiUrl}
-        value={settings?.url}
-        placeholder={"https://vectorapi.grafana.com"}
-        onChange={e => onChange({ ...settings, url: e.currentTarget.value })}
-      />
-    </Field>
+    <>
+      <InlineField>
+        <LegacyForms.FormField
```

Comment:
Why the switch to a legacy form field? 🤔

Reply:
Copied the logic from auth in datasources. Any suggestion on what to use, otherwise maybe we can drop it back to an input field.

Reply:
Maybe just an input field tbh 🤷